### ISSUE TITLE
Problem displaying nested models

### ISSUE THREAD

**Issue by maksymgendin** [smoketurner/dropwizard-swagger#95]
Hi all!

I have problems with the new Swagger UI. It is not able to properly display nested models, neither in `Example Value` nor in `Model` tab in my `Responses`. The nested model is just empty there. But the nested model is properly listed in the `Models` block on the bottom of the page.
The `swagger.json` file is valid too, it shows me the correct reference in `responses`, like `"schema":{"$ref":"#/definitions/ErrorResponse"}`...and when I then take a look at `#/definitions/ErrorResponse` I see there in `properties` my nested `validationErrors` field with `type: "array"` and the reference to the `ValidationError` model `$ref:"#/definitions/ValidationError"`...but Swagger UI is not able to show me the structure of my `ValidationError` inside `ErrorResponse`...

Someone faced that problem too?

Best regards
Maksym

**Comment by jplock** [comment]
Is this an issue with this project or swagger-ui?

**Comment by maksymgendin** [comment]
I'm not sure yet...it worked for me with version `1.1.2-1` and Swagger UI `2.2.10`...I will try to test my case on detect the source of my issue 😄

**Comment by maksymgendin** [comment]
So, https://editor.swagger.io is able to correctly display me this Swagger YAML file:

```
---
swagger: "2.0"
info:
  title: "test"
  version: "1.0"
tags:
- name: "User"
schemes:
- "http"
paths:
  /user/{userId}:
    get:
      tags:
      - "User"
      operationId: "get"
      parameters:
      - name: "userId"
        in: "path"
        required: true
        type: "string"
        format: "uuid"
      responses:
        200:
          description: "successful operation"
          schema:
            $ref: "#/definitions/UserGet"
          headers: {}
definitions:
  UserGet:
    type: "object"
    required:
    - "id"
    properties:
      id:
        type: "string"
        format: "uuid"
        example: "2e79bb36-8255-4b3e-beb6-a2af03e20c6e"
      emails:
        type: "array"
        uniqueItems: true
        items:
          $ref: "#/definitions/UserGetEmail"
  UserGetEmail:
    type: "object"
    required:
    - "email"
    - "id"
    properties:
      id:
        type: "string"
        format: "uuid"
        example: "2e79bb36-8255-4b3e-beb6-a2af03e20c6e"
      email:
        type: "string"
        minLength: 0
        maxLength: 255
```

`Example Value` and `Model` tabs are correct in `Responses` of `GET /user/{userId}` and the model on the bottom is also well structured. But if I use `dropwizard-swagger`, it shows me in `Example Value`:

```
{
  "id": "2e79bb36-8255-4b3e-beb6-a2af03e20c6e",
  "emails": [
    null
  ]
}
```
And the model looks like that https://pasteboard.co/GYbH0lN.png

**Comment by maksymgendin** [comment]
@jplock Any thoughts?

**Comment by maksymgendin** [comment]
@jplock Seems to be fixed now in version `1.2.2-2`
